### ISSUE TITLE
pubsub on MacOs does not properly terminate react app

### ISSUE THREAD

**Issue by dmitsh** [dapr/quickstarts#522]
## Actual Behavior
After successfully completion of the pubsub quickstart on MacOS, the react app is not properly terminated.

```bash
% lsof -PiTCP -sTCP:LISTEN
COMMAND     PID USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
node      25269 dima   21u  IPv6 0x5e0941cf3c20a035      0t0  TCP *:8080 (LISTEN)

% ps -aef | grep 25269
  501 25269     1   0  3:05PM ttys001    0:00.33 node server.js
```

## Steps to Reproduce the Problem

Run the following steps on MacOS

1. Run the pubsub quickstart: `mm.py ./README.md`
2. Once completed, check the listening ports

**Comment by wcs1only** [comment]
Think this is a CLI bug that we already have an issue for: https://github.com/dapr/cli/issues/695.

TL;DR: Cli sends SIGKILL instead of SIGTERM. Node spawns a parent and a child process. The parent gets a SIGKILL from the CLI when it exits, which doesn't allow the parent to clean up the child, the child is leaked.

**Comment by paulyuk** [comment]
@artursouza could you please recommend who should have a look?  This impacts function and test health.

**Comment by mukundansundar** [comment]
@paulyuk @pkedy  was working on the PR for the fix for the dapr/cli#695. I have pinged him on that PR if he will pick it up.

**Comment by amulyavarote** [comment]
Closing this due to inactivity. Please feel free to reopen this issue if needed.

**Comment by amulyavarote** [comment]
Closing this issue due to inactivity. Please feel free to reopen this if needed.
